src/barkhaPJun20/Bank.java

```diff
@@ -0,0 +1,88 @@
+package barkhaPJun20;
+
+public class Bank {
+	int debitCount=0;
+	int creditCount=0;
+	int balanceCount=0;
+	
+	static int overallDebitCount=0;
+	static int overallCreditCount=0;
+	static int overallBalanceCount=0;
```

Comment:
Remove the initialization of all instance variables and remove extra space.

Reply:
Done